File: ui/vzk-ui/src/app/models/result/result.ts
export class Result {
  id: string;
  title: string;
  game: string;
  dateTime: Date;
  description: string;
  enemyLogoURL: string;
  enemyResult: string;
  teamResult: string;
  replayURL: string;

  constructor(id: string, title: string, datetime: Date, description: string, game: string, enemyURL: string, enemyResult: string, teamResult: string,   replayURL: string) {
    this.id = id;
    this.title = title;
    this.dateTime = datetime;
    this.description = description;
    this.game = game;
    this.enemyLogoURL = enemyURL
    this.enemyResult = enemyResult;
    this.teamResult = teamResult;
    this.replayURL = replayURL;
  }
}

File: events/src/main/java/com/vzk/events/models/Result.java
package com.vzk.events.models;

import jakarta.persistence.*;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import java.util.UUID;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Entity(name = "result")
public class Result {
    @Id
    @Column(name = "id")
    private UUID id;

    @Basic
    @Column(name = "title")
    private String title;

    @Basic
    @Column(name = "game")
    private UUID game;

    @Basic
    @Column(name = "date_time")
    private String dateTime;

    @Basic
    @Column(name = "enemy_logo_url")
    private String enemyLogoURL;

    @Basic
    @Column(name = "enemy_result")
    private String enemyResult;

    @Basic
    @Column(name = "team_result")
    private String teamResult;

    @Basic
    @Column(name = "replay_url")
    private String replayURL;

    @Basic
    @Column(name = "is_active")
    private boolean isActive;

    @Basic
    @Column(name = "description")
    private String description;

    @OneToOne(cascade = CascadeType.ALL)
    @JoinColumn(name = "event_id", referencedColumnName = "id")
    private Event event;
}
